source-connector/src/test/kotlin/org/neo4j/connectors/kafka/source/Neo4jCdcSourceRelationshipsIT.kt

```diff
@@ -129,35 +117,23 @@ abstract class Neo4jCdcSourceRelationshipsIT {
                           patterns = arrayOf(CdcSourceParam("()-[:RELIES_TO {}]->()"))))))
   @Test
   fun `should read property removal and additions`(
-      testInfo: TestInfo,
       @TopicConsumer(topic = "neo4j-cdc-rels-prop-remove-add", offset = "earliest")
       consumer: ConvertingKafkaConsumer,
       session: Session
   ) {
-    val executionId = testInfo.displayName + System.currentTimeMillis()
-    val params = mapOf("execId" to executionId)
     session
         .run(
-            """CREATE (s:TestSource {name: 'Bob', execId: ${'$'}execId})
-            |CREATE (t:TestSource {name: 'Alice', execId: ${'$'}execId})
-            |CREATE (s)-[:RELIES_TO {weight: 1, rate: 42, execId: ${'$'}execId}]->(t)
+            """CREATE (s:TestSource {name: 'Bob'})
+            |CREATE (t:TestSource {name: 'Alice'})
+            |CREATE (s)-[:RELIES_TO {weight: 1, rate: 42}]->(t)
     """
-                .trimMargin(),
-            params)
-        .consume()
-    session
-        .run(
-            "MATCH (:TestSource)-[r:RELIES_TO {execId: \$execId}]-(:TestSource) SET r.weight = 2, r.rate = NULL",
-            params)
-        .consume()
-    session
-        .run(
-            "MATCH (:TestSource)-[r:RELIES_TO {execId: \$execId}]-(:TestSource) SET r.rate = 50",
-            params)
+                .trimMargin())
         .consume()
     session
-        .run("MATCH (:TestSource)-[r:RELIES_TO {execId: \$execId}]-(:TestSource) DELETE r", params)
+        .run("MATCH (:TestSource)-[r:RELIES_TO]-(:TestSource) SET r.weight = 2, r.rate = NULL")
```

Comment:
`SET r.rate = NULL` is allowed? I thought we had to run `REMOVE r.rate`

Reply:
Yep, it's basically equivalent to removing a property.